### PR TITLE
rename `secret:bulk` top-level command to `secret bulk` subcommand

### LINKED ISSUE #5168: Remove ':' from secret:bulk command
### Describe the solution

Deprecate – but keep – the existing secret:bulk command

Rename to something consistent with other commands similar to https://github.com/cloudflare/workers-sdk/issues/5062

### PR DESCRIPTION
**Pull request by RamIdeas**
and deprecate `secret:bulk` – command still exists with same functionality but also with a deprecation notice

Fixes #5168

## What this PR solves / how to test

Fixes #[insert GH or internal issue number(s)].

## Author has addressed the following

- Tests
  - [ ] TODO (before merge)
  - [x] Included
  - [ ] Not necessary because:
- E2E Tests CI Job required? (Use "e2e" label or ask maintainer to run separately)
  - [ ] I don't know
  - [ ] Required / Maybe required
  - [x] Not required because: unit tested
- Changeset ([Changeset guidelines](https://github.com/cloudflare/workers-sdk/blob/main/CONTRIBUTING.md#changesets))
  - [ ] TODO (before merge)
  - [x] Included
  - [ ] Not necessary because:
- Public documentation
  - [x] TODO (before merge)
  - [ ] Cloudflare docs PR(s): 
  - [ ] Not necessary because:

### CODE REVIEW

**Comment by CarmenPopoviciu** [comment]
should we mark https://github.com/cloudflare/workers-sdk/blob/main/packages/wrangler/src/index.ts#L399-L404 as `DEPRECATED` with a comment? like we do for other deprecated cmds?

File: packages/wrangler/src/__tests__/secret.test.ts [hunk 1]
```diff
@@ -745,6 +766,13 @@ describe("wrangler secret", () => {
 
 			"
 		`);
+			expect(std.warn).toMatchInlineSnapshot(`
+			"[33m▲ [43;33m[[43;30mWARNING[43;33m][0m [1m\`wrangler secret:bulk\` is deprecated and will be removed in a future major version.[0m
+
+			  Please use \`wrangler secret bulk\` instead, which accepts exactly the same arguments.
```
Comment (andyjessop) [inline packages/wrangler/src/__tests__/secret.test.ts:772]: Not for this PR of course, but I'm wondering if it's worth using an abstraction to remove the boilerplate from these snapshots. I suppose the pro is that it's better to be explicit about the output, and the cons are that it's less readable and potentially less maintainable.

**Review by andyjessop — APPROVED** [review]
Looks good! KV next..?

**Review by CarmenPopoviciu — APPROVED** [review]

File: packages/wrangler/src/__tests__/secret.test.ts (hunk 1 above)
Comment (RamIdeas) [reply packages/wrangler/src/__tests__/secret.test.ts:772]: In this case, this was intentional to test the deprecation message shows. But as mentioned on chat, I'm going to make that a bit more explicit like in https://github.com/cloudflare/workers-sdk/pull/5989

**Pull request merged**